_includes/selector.html

```diff
@@ -484,7 +460,7 @@
                     pkgs = ["rapids"];
                 }
 
-                var pkgs_vers = pkgs.map(pkg => this.highlightPkgOrImg(pkg) + "=" + rapids_version + " ").join("").toLowerCase();
+                var pkgs_vers = pkgs.filter(pkg => pkg !== "Choose Specific Packages").map(pkg => this.highlightPkgOrImg(pkg) + "=" + rapids_version + " ").join("").toLowerCase();
```

Comment:
Maybe this can be a way of enforcing at least one package? The empty install command is awkward imo.
```suggestion
                if ((pkgs.length + this.active_additional_packages.length) == 0) {
                    return "Select at least one package.";
                }
                var pkgs_vers = pkgs.filter(pkg => pkg !== "Choose Specific Packages").map(pkg => this.highlightPkgOrImg(pkg) + "=" + rapids_version + " ").join("").toLowerCase();
```